lib/repository.py

```diff
@@ -48,9 +48,27 @@ def get_git_repository(remote_repo_url, parent_dir_path):
     name = os.path.basename(os.path.splitext(url_parts.path)[0])
 
     repo_path = os.path.join(parent_dir_path, name)
-    if os.path.exists(repo_path):
-        MAIN_REMOTE_NAME = "origin"
+    try:
         repo = GitRepository(repo_path)
+    except git.exc.InvalidGitRepositoryError:
+        # This exception will be thrown if the Git repository has an
+        # invalid format, which could be due to the repository
+        # changing from SVN to Git, for example.  A way to proceed is
+        # removing the directory and cloning from scratch.
+        message = ("Repository path '{repo_path}' has invalid format. "
+                   "If you are unsure, remove it and try again."
+                   .format(repo_path=repo_path))
+        LOG.exception(message)
```

Comment:
`LOG.exception` will print the stack trace. Since you're re-raising the exception, it should be printed anyway outside of here (possibly with the program abortion), so it's better to use `LOG.error` to avoid duplicating it.